server/db/utils.js

```diff
@@ -4,11 +4,16 @@
 
 'use strict';
 
+var createHash = require('crypto').createHash;
+
 var mysql = require('mysql');
 var config = require('../config');
 var log = require('../logger')('server.db.utils');
 
 module.exports = {
+  createUrlHash: function (url) {
+    return createHash('sha1').update(url).digest('hex').toString();
+  },
```

Comment:
This snippet was used 3x in the files above, so I refactored it out here into the `utils.js` file. Plus, makes it mildly easier to tweak the URL hashing method in the future if we want to.